AM_PATH_LIBOTR(...) fails to find libotr even if it is correctly
installed.

XXX: Investigate this later with @olabini
--- configure.ac.orig	2019-04-14 09:27:27 UTC
+++ configure.ac
@@ -71,7 +71,7 @@ dnl Checks for libraries.
 PKG_CHECK_MODULES([GLIB], [glib-2.0 >= 2.18])
 PKG_CHECK_MODULES([LIBGOLDILOCKS], [libgoldilocks >= 0.0.1])
 PKG_CHECK_MODULES([LIBSODIUM], [libsodium >= 1.0.0])
-AM_PATH_LIBOTR(4.0.0,,AC_MSG_ERROR(libotr 4.x >= 4.0.0 is required.))
+PKG_CHECK_MODULES([LIBOTR], [libotr >= 4.0.0])
 AM_PATH_LIBGCRYPT(1:1.6.0,
   [AC_DEFINE([HAVE_GCRYPT], [1], [Use GCRYPT])],
   AC_MSG_ERROR(libgcrypt 1.6.0 or newer is required.)
